### PR TITLE
lib: make cockpit-logs-panels independent from PF3

### PR DESCRIPTION
**Pull request by KKoukiou**
The badges look different in bootstrap compared to PF4. Are we happy with the change?

Before:
Screen Shot 2021-03-10 at 14 59 31

After:
Screen Shot 2021-03-10 at 14 59 20

### CODE REVIEW

**Comment by marusak** [comment]
> Are we happy with the change?

I am :) But I am not the one to make this decision

**Comment by garrett** [comment]
Wow, this is much needed! Looks great in the screenshots!

I'm building it now and will approve it immediately if I don't find anything unexpected.

**Review by garrett — DISMISSED** [review]
Looks good locally! :+1:

I'll check the livetest in a minute too, but based on screenshots and local testing, it's good!

**Comment by martinpitt** [comment]
Sorry about the "Aborted without status", I had to restart our bots. But there are plenty of finished tests, and the [journal test still fails](https://logs.cockpit-project.org/logs/pull-15499-20210310-142035-7766ffdd-debian-stable/log.html#283-2)

File: pkg/lib/cockpit-components-logs-panel.jsx
Comment (marusak) [inline pkg/lib/cockpit-components-logs-panel.jsx:86]: Do we need that `>`?
I never understood what it was trying to say.
For example we have the same logic of "squash same reports" in selinux and we don't show that icon there:
counter

Comment (KKoukiou) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: The difference between selinux and logs panel is that in the logs that badge is actually redirecting to the logs details when clicked on. I agree that's not obvious at all though that this arrow suggests that the badge contains a link.

Comment (KKoukiou) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: To be clear, it's not the badge that it's redirecting, the whole line is having the onClick handlers.

Comment (KKoukiou) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: Heh, we commented together. Shall I repost then removing the arrows and making the onClick be on the text?

Comment (marusak) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: @garrett WDYT
- is that arrow necessary? (This predates me, so I am not sure, if there has been some reason behind this)
- Would it be fine, if only text would redirect and not the whole line? Now on hover I think just the text changes color and cursor.

Comment (KKoukiou) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: @marusak How about I post a followup PR on this? I have the code change locally in fact, but since this is fine from @garrett and the tests seem happy I would rather merge and introduce the design change in a seperate PR.

**Review by marusak — APPROVED** [review]
Thank you!

File: pkg/lib/cockpit-components-logs-panel.jsx
Comment (marusak) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: Agree! Lets not block on this, especially since this blocks another PR.

Comment (KKoukiou) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: I see an issue with making the message the only clickable part of a row:

* If we make it a link or a button it will become blue by default (probably we don't want that)
* If we don't make it link or a button it will not be blue and the users will not have any idea where to click on the row to get redirected. Now indeed we only underline the message on hover, but at least clicking anywhere in the row will redirect.

Comment (garrett) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: I don't think the arrow is necessary. 

It doesn't do anything. From my understanding, the number is just the count of the number of the times it is repeated. I never understood why it had an arrow. (It predates me working on Cockpit too.)

If we all agree it isn't necessary, let's just drop it?

Comment (marusak) [reply pkg/lib/cockpit-components-logs-panel.jsx:86]: Agree! We will do this as follow up though as we want to move this forward.
@KKoukiou will you send PR for that?

**Pull request merged**
